tests/framework/behaviors/AttributeBehaviorTest.php

```diff
@@ -0,0 +1,139 @@
+<?php
+
+namespace yiiunit\framework\behaviors;
+
+use Yii;
+use yiiunit\TestCase;
+use yii\db\Connection;
+use yii\db\ActiveRecord;
+use yii\behaviors\AttributeBehavior;
+
+/**
+ * Unit test for [[\yii\behaviors\AttributeBehavior]].
+ * @see AttributeBehavior
+ *
+ * @group behaviors
+ */
+class AttributeBehaviorTest extends TestCase
+{
+    /**
+     * @var Connection test db connection
+     */
+    protected $dbConnection;
+
+    public static function setUpBeforeClass()
+    {
+        if (!extension_loaded('pdo') || !extension_loaded('pdo_sqlite')) {
+            static::markTestSkipped('PDO and SQLite extensions are required.');
+        }
+    }
+
+    public function setUp()
+    {
+        $this->mockApplication([
+            'components' => [
+                'db' => [
+                    'class' => '\yii\db\Connection',
+                    'dsn' => 'sqlite::memory:',
+                ]
+            ]
+        ]);
+
+        $columns = [
+            'id' => 'pk',
+            'name' => 'string',
+            'alias' => 'string',
+        ];
+        Yii::$app->getDb()->createCommand()->createTable('test_attribute', $columns)->execute();
+    }
+
+    public function tearDown()
+    {
+        Yii::$app->getDb()->close();
+        parent::tearDown();
+    }
+
+    // Tests :
+
```

Comment:
can you wrap the test methods into 1 test method with a dataprovider? This enables code-reuse and its easier to compare the data used for testing.